lib/chef/provider/service.rb

```diff
@@ -26,6 +26,9 @@ class Service < Chef::Provider
 
       include Chef::Mixin::Command
 
+      # depends on 'require mixlib' from above 'include mixin::command'
+      TIMEOUT = Mixlib::ShellOut::DEFAULT_READ_TIMEOUT
```

Comment:
if you default to nil, you can make it pick up mixlib-shellout's timeout and avoid this.